internal/abci/abci.go

```diff
@@ -623,7 +623,7 @@ type indexedTxn struct {
 //
 // NOTE: This is a plain function instead of an AbciApplication method so that
 // it may be directly tested.
-func (a *AbciApp) prepareMempoolTxns(txs [][]byte, maxBytes int, log *log.Logger, proposerAddr []byte) ([][]byte, uint64) {
+func (a *AbciApp) prepareMempoolTxns(txs [][]byte, log *log.Logger, maxTxBytes int64, proposerAddr []byte) [][]byte {
```

Comment:
Should update docs for this method, and probably rename it from `prepareMempoolTxns` since it now injects the proposer transactions too, which used to be done in `PrepareProposal`